src/main/java/org/llorllale/cactoos/matchers/Throws.java

```diff
@@ -104,15 +161,19 @@ protected boolean matchesSafely(final Scalar<T> obj,
      *
      * @param dsc The description of the object.
      * @param type The exception type.
-     * @param msg The exception message.
+     * @param expectationDescription The description of the message
+     *  expectation.
      */
-    private static void describe(final Description dsc, final Class<?> type,
-        final String msg) {
+    private static void describe(
+        final Description dsc,
+        final Class<?> type,
```

Comment:
@vzurauskas the same, see parameter naming limitations above

Reply:
@ilyakharlamov Changed the name to `exception`.